### ISSUE TITLE
wrong Config.t length test line 293 with inc_version_list_init

### ISSUE THREAD

**Issue by rurban** [perl11/p5-Config#9]
eg: inc_version_list_init='"5.22.0","5.21.11","5.21.10","5.21.9","5.21.8","5.21.5","5.21.4","5.21.3","5.21.2","5.21.1","5.20.3","5.20.2","5.20.1","5.20.0","5.19.9","5.19.8","5.19.6","5.19.4","5.19.2","5.18.4","5.18.2","5.18.1","5.18.0","5.16.3","5.16.1","5.15.8","5.14.4","5.14.3","5.14.2","5.14.1","5.14.0","5.12.5","5.12.4","5.10.1","5.8.9","5.8.8","5.8.5","5.8.4","5.6.2",0'

will cause this test to fail.
fixed with 

```
--- ext/Config/t/Config.t
+++ ext/Config/t/Config.t
@@ -290,7 +290,7 @@ foreach my $lib (qw(applibexp archlibexp privlibexp sitearchexp sitelibexp
         my $v = $Config{$k};
         if (defined $v and length $v
             and $v =~ /^".*?"(.*)/
-            and $k ne 'sig_name_init')
+            and $k !~ /(sig_name|list)_init$/)
         {
             ok !length $1, "$k => $v";
         }
```

**Comment by rurban** [comment]
See https://github.com/perl11/cperl/commit/70282c76cf864d9a1fc24cfd484e5192e79246e0

**Comment by bulk88** [comment]
Shipped in 6.07.
